scripts/build/release.sh

```diff
@@ -181,21 +181,20 @@ touch $CHANGELOG_FILE
 # Do the git stuff only if we are in develop branch
 #
 CURRENT_BRANCH=$(git branch | grep '^*' | cut -c 3-10)
-if [ "$CURRENT_BRANCH" == "develop" ]
+if [ "$CURRENT_BRANCH" == "master" ]
 then
     git add rpm/SPECS/pepProxy.spec
     git add rpm/create-rpm.sh
     git add package.json
     git add CHANGES_NEXT_RELEASE
     git commit -m "ADD Step: $currentVersion -> $NEW_VERSION"
-    git push origin develop
+    git push origin master
 
     # We do the tag only and merge to master only in the case of  non "dev" release
     if [ "$PEP_RELEASE" = "sprint" ]
     then
        git checkout master
        git pull origin master
-       git merge develop
```

Comment:
You are already in master and you have just pushed master to origin, so lines 196 and 197 could be also removed (see https://github.com/telefonicaid/perseo-fe/pull/194/files#diff-db01dd9c2e5c291a004d2e3d3c73bff6L201)